### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dilipchauhan14**
Removed Useless Comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,3 @@ Helps you to waste your time.
 #readme file altered
 
 # webhooks payload url https://s9u0wdufd0.execute-api.us-east-2.amazonaws.com/default/github_pull_request
-
```
Comment (dilipchauhan14) [inline README.md]: Blank Line

File: README.md [hunk 2]
```diff
@@ -7,15 +7,3 @@ Helps you to waste your time.
...
-ASDFGHJKHJKL
```
Comment (dilipchauhan14) [inline README.md]: Meaning less alphabets.

File: README.md [hunk 3]
```diff
@@ -7,15 +7,3 @@ Helps you to waste your time.
...
-ASDFGHJKL
-asdfghjkl
-ABC
```
Comment (dilipchauhan14) [inline README.md]: kkbkb

File: README.md (hunk 1 above)
Comment (sammarth-kapse) [inline README.md]: Good Work

**Pull request merged**